Library/Homebrew/api/cask.rb

```diff
@@ -23,12 +23,20 @@ def fetch_source(token, git_head: nil)
         sig { returns(Hash) }
         def all_casks
           @all_casks ||= begin
-            json_casks = Homebrew::API.fetch_json_api_file "cask.json",
-                                                           target: HOMEBREW_CACHE_API/"cask.json"
+            json_casks, updated = Homebrew::API.fetch_json_api_file "cask.json",
+                                                                    target: HOMEBREW_CACHE_API/"cask.json"
 
-            json_casks.to_h do |json_cask|
+            casks = json_casks.to_h do |json_cask|
               [json_cask["token"], json_cask.except("token")]
             end
+
+            cask_names_path = HOMEBREW_CACHE_API/"parsed/cask_names.txt"
```

Comment:
```suggestion
            cask_names_path = HOMEBREW_CACHE_API/"cask_names.txt"
```

Reply:
What do you suggest we do for the `update.sh` part that removes the directory. Remove *.txt?

Reply:
Or just list the 3 files to remove maybe.

Reply:
@Bo98 yeh, either seems fine.